src/com/cft/contactmerge/io/SplitStringUtilities.java

```diff
@@ -10,23 +10,18 @@
 import java.util.*;
 
 public class SplitStringUtilities {
-    public static List<String> splitTsvString(String input)
-    {
+    public static List<String> splitTsvString(String input) {
         List<String> x = new ArrayList<String>();
 
-
-
         String testString = input;
 
-
         String _string = new String("");
         for(int i = 0;i< testString.length();i++){
```

Comment:
If this is the only place `testString` is being called, wouldn't it be better to just replace this with `input`? This will save the program from making an unneeded instantiation.